doc/api/n-api.md

```diff
@@ -4612,7 +4616,8 @@ napi_create_threadsafe_function(napi_env env,
 ```
 
 - `[in] env`: The environment that the API is invoked under.
-- `[in] func`: The JavaScript function to call from another thread.
+- `[in] func`: An Optional JavaScript function to call from another thread.
+If `call_js_cb` is not given, type check would be enforced on the `func`.
```

Comment:
How about "It must be provided if `NULL` is passed to `call_js_cb`."?

Reply:
@legendecas please also modify the documentation for `napi_threadsafe_function_call_js` to indicate that `js_cb` may be NULL if the thread-safe function was created without a `func`.

Reply:
Added an additional statement on the condition of NULL js_cb :)

Reply:
> How about "It must be provided if `NULL` is passed to `call_js_cb`."?

Nit remains.